.github/workflows/ci.yml

```diff
@@ -87,19 +92,17 @@ jobs:
           restore-keys: audit_3.1
 
       - name: Verify Patch Coverage
-        working-directory: source/audit_report/3.1.0
-        run: python3 -m audit unrefed --yaml changes
+        working-directory: source/docs/audit_report
+        run: poetry run python3 -m genaudit.cli unrefed --yaml changes
         env:
-          PYTHONPATH: ${{ github.workspace }}/source/audit_generator
           AUDIT_CACHE_LOCATION: ${{ github.workspace }}/audit_generator_cache
           AUDIT_REPO_LOCATION: ${{ github.workspace }}/botan
           BASIC_GH_TOKEN: ${{ github.token }}
 
       - name: Build Document
-        working-directory: source/audit_report/3.1.0
-        run: make latexpdf
+        working-directory: source/docs/audit_report
+        run: poetry run make latexpdf
         env:
-          PYTHONPATH: ${{ github.workspace }}/source/audit_generator
```

Comment:
No more nitty-gritty python dependency environment setup in the top-level CI. This information is now kept in the `pyproject.toml` of the audit_report document and the setup is taken care of by `poetry`.